src/components/Contentful/DynamicHero.vue

```diff
@@ -79,28 +97,49 @@
 <script>
 import KvButton from '@/components/Kv/KvButton';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import KvCarousel from '@/components/Kv/KvCarousel';
```

Comment:
Could these be dynamically imported depending on `isHeroCarousel` ?

Reply:
Yeah possibly? I can  do that in a followup PR. Are you concerned about loading embla and carousel when it's not needed?

Reply:
Yeah, exactly. Embla has some weight. More broadly working towards dynamically importing only the components in use when we have conditional logic like experiments and Contentful driven stuff.

Reply:
I also realize that Embla is currently loaded everywhere in our vendors bundle right now, but we should be able to sort that out separately.